### ISSUE TITLE
Export Dino

### ISSUE THREAD

**Issue by lettucetrail** [cadon/ARKStatsExtractor#1219]
Is it possible to export a dino from your library in the same format as the export files from Ark?

When I have issues with extracting (As I currently am with a mutgen'd megalo) I sometimes download my world save and save it to a temporary library. I'd like to simply take the one dino I want from that temporary library and save it to my main library. I don't see a way to do that in app.

**Comment by lettucetrail** [comment]
Huh. I kinda assumed I'd be able to properly adjust the values manually using the information I have from the other library. But I can't seem to manually add a dinosaur with stats that I know are correct.

**Comment by cadon** [comment]
You can just enter the levels in ther `Tester` tab and add the creature.

If you applied mutagen to a creature, a bug in ARK prevents the torpidity value to be set, so the outdated and wrong value is shown, this prevents the creature to be imported. To update the value, cryo and uncryo the creature you put mutagen in, then you can import it.

There's theoretically a way to copy & paste a creature, but I just tested that and it's not working.

**Comment by lettucetrail** [comment]
Yeah I've attempted cryoing/uncryoing, I've tried uploading and downloading, the server should have even been updated so it should have restarted. 

There's a possibility it's an issue with my configs of course. But no matter what I can't seem to get it to read the extraction. My gut instinct says it's something to do with my configuration? Most of the things I've encountered with this program have been my user error so I imagine that's the case here. Like how I didn't know the stat tester tab existed.

**Comment by cadon** [comment]
If multipliers are the issue and the creature is bred, it's most likely the `BabyImprintingStatScaleMultiplier`.
